Update default bootstraprc-4 properties to reflect changes in bootstrap v4 alpha 6

mircowidmer opened the pull request:
- renamed 'animation' to 'transitions' and 'tags' to 'badge' (fixes #292)
- ordered default properties alphabetically
- updated example configs

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/shakacode/bootstrap-loader/293)
<!-- Reviewable:end -->

apawsey commented:
You beat me to this :)
There are at least a couple more changes to get everything alpha6-ified...
Need to make the same changes to the examples configs, and the useFlexbox config can be dropped.

mircowidmer commented:
@apawsey You are right about the dropping of useFlexbox, but I wanted to make this change only about the renaming of the two properties ;)

justin808 commented:
Once we have a few people verify that this PR works, I'm happy to merge and release!

justin808 commented:
@mircowidmer @apawsey Can both of you confirm that all the examples work.

Please also update:

CHANGELOG.md
README.md

justin808 commented:
@mircowidmer Any ETA?

justin808 commented:
@mircowidmer This looks great. Can you please

1. Confirm that you tested all the examples in /examples
2. Confirm tests and lints are passing
3. Create a changelog.md entry

---

Reviewed 1 of 1 files at r1, 3 of 3 files at r2, 3 of 3 files at r3.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/shakacode/bootstrap-loader/293#-:-KiJsPD091TnSrvhwXui:bpsu3qf)*
<!-- Sent from Reviewable.io -->

mircowidmer commented:
@justin808 

1. I tested the examples `basic` and `css-modules` (`npm run bs3`, `npm run bs4`)
2. I tested the example `multiple-entries` only with `npm run build`
3. `npm run test` and `npm run lint` are passing
4. In the last commit I added a changelog entry and updated the readme

One comment: `npm run bs4` in the basic example failed somewhat due to missing jQuery ("Uncaught ReferenceError: jQuery is not defined"). I could reproduce the same error before my changes: https://github.com/shakacode/bootstrap-loader/commit/3347eb376b7e1c0807f8dd4bb3c219fbdec3ae7c

I am not sure about updating the version to 3.0.0 but since it is a breaking change (e.g. the bootstrap dependency has to be updated), that seemed reasonable to me.

justin808 commented:
@mircowidmer I have not been updating the major version number for alpha/beta updates since it's sort of understood that those will be breaking. Also, our changes should not affect bootstrap 3 users, correct?

mircowidmer commented:
So what version number are you suggesting then, I'll happily edit my last commit then. As far as I can tell, bootstrap 3 users should not be affected, yes.

justin808 commented:
@tc-imba -- can you confirm this one is working.
@mircowidmer -- yes no major version bump

justin808 commented on CHANGELOG.md:
```diff
@@ -3,7 +3,9 @@ All notable changes to this project will be documented in this file. Items under
 Contributors: please follow the recommendations outlined at [keepachangelog.com](http://keepachangelog.com/). Please use the existing headings and styling as a guide, and add a link for the version diff at the bottom of the file. Also, please update the `Unreleased` link to compare to the latest release version.
 
-## Unreleased
+## [3.0.0] - ???
```
@mircowidmer this should be 2.1.0

mircowidmer commented:
@justin808 I updated README and CHANGELOG and bumped version within package.json to 2.1.0.
